x/concentrated-liquidity/swaps.go

```diff
@@ -340,11 +375,22 @@ func (k Keeper) calcOutAmtGivenIn(ctx sdk.Context,
 				return writeCtx, sdk.Coin{}, sdk.Coin{}, sdk.Int{}, sdk.Dec{}, sdk.Dec{}, err
 			}
 		}
+
+		// update the swapState with the new sqrtPrice from the above swap
+		swapState.sqrtPrice = sqrtPrice
+		// we deduct the amount of tokens we input in the computeSwapStep above from the user's defined tokenIn amount
+		swapState.amountSpecifiedRemaining = swapState.amountSpecifiedRemaining.Sub(amountIn.Add(feeChargeTotal))
+		// we add the amount of tokens we received (amountOut) from the computeSwapStep above to the amountCalculated accumulator
+		swapState.amountCalculated = swapState.amountCalculated.Add(amountOut)
```

Comment:
Oh interesting, this looks like something we should have had prior to this PR as well, do we not have test cases for testing if these values were properly returned? If not, should we create an issue to track that?

Reply:
These are just moved from a few lines above. No changes have been made other than moving

Reply:
Also, these are tracked here:
- https://github.com/osmosis-labs/osmosis/issues/4089
- https://github.com/osmosis-labs/osmosis/issues/4086

Reply:
I moved this in the location as to reduce diff to make this less confusing